chall05/vsempere.c

```diff
@@ -0,0 +1,61 @@
+# include <stdlib.h>
+
+static size_t		ft_strlen(const char *s)
+{
+	size_t	i;
+
+	i = 0;
+	while (s[i] != 0)
+		i++;
+	return (i);
+}
+
+static char		*ft_strstr(const char *big, const char *little)
+{
+	size_t		i;
+	size_t		j;
+
+	i = 0;
+	j = 0;
+	while (big[i])
+	{
+		while (big[i + j] == little[j] && big[i + j])
+			j++;
+		if (little[j] == '\0')
+			return ((char *)(big + i));
+		j = 0;
+		i++;
+	}
+	return (NULL);
+}
+
+static char ft_check_exception(char *str)
+{
+	char	*exc;
+	char	*f;
+	size_t	strl;
+
+	exc = "sleigh stein fahrenheit deifies either nuclei reimburse ancient juicier societies";
+	if (!(f = ft_strstr(exc, str)))
+		return (0);
+	strl = ft_strlen(str);
+	return (((f > exc && *(f - 1) == ' ') || f == str) && (f[strl] == '\0' || f[strl] == ' ') ? 1 : 0);
+}
```

Comment:
Mmmh the exercice is loosing its point if you return true when an exception is present.  
You are also forgetting most of the exceptions... what about `eight`, for example?

By using this function, you are failing the chall 😅